src/common-util/src/test/java/ee/ria/xroad/common/util/CertUtilsTest.java

```diff
@@ -118,4 +120,50 @@ public void testCreatePkcs12() throws Exception {
         Path path = Paths.get(pkcsPath);
         assertTrue(Files.exists(path));
     }
+
+    @Test
+    public void testGetSubjectAlternativeNamesEmpty() throws CertificateException, IOException {
+        final String certPath = "src/test/resources/cert_empty.pem";
+        CertificateFactory cf = CertificateFactory.getInstance("X.509");
+        try (FileInputStream in = new FileInputStream(certPath)) {
+            X509Certificate cert = (X509Certificate) cf.generateCertificate(in);
+            assertEquals(null, CertUtils.getSubjectAlternativeNames(cert));
+        }
+    }
+
+    @Test
+    public void testGetSubjectAlternativeNamesSimple() throws CertificateException, IOException {
+        final String certPath = "src/test/resources/cert_simple.pem";
+        CertificateFactory cf = CertificateFactory.getInstance("X.509");
+        try (FileInputStream in = new FileInputStream(certPath)) {
+            X509Certificate cert = (X509Certificate) cf.generateCertificate(in);
+            assertEquals("DNS:*.dev.acme.org", CertUtils.getSubjectAlternativeNames(cert));
```

Comment:
It is a good practice to use IANA reserved example domain names in documentation and tests (see e.g. https://tools.ietf.org/html/rfc2606#section-3).